backend/test_observer/controllers/users/users.py

```diff
@@ -11,7 +11,7 @@
 
 
 # include_in_schema=False to hide this endpoints from docs as its internal use
-@router.post("/", response_model=UserDTO, include_in_schema=False)
+@router.post("", response_model=UserDTO, include_in_schema=False)
```

Comment:
sorry if it's a dumb question, but isn't `/` correct?

Reply:
It's correct, but just to be consistent with other endpoints. The reasoning is described [here](https://github.com/canonical/test_observer/blob/main/backend/test_observer/main.py#L34-L39)